components/input.css

```diff
@@ -0,0 +1,33 @@
+
+ .input{
+     width: 80%;
+     padding: 10px;
+     height: 1rem;
+  }
+.input_bar {
+    display: flex;
+    flex-direction: row;
+    width: 80%;
+    background-color: var(--offline-color);
+    padding: 10px;
+    margin: 10px;
+    align-items: center;
+    justify-content: space-around;
+   border-radius: 10px;
```

Comment:
The border-radius is being applied on the div and not on the input box

Reply:
+1

Reply:
thanks updated